### PR TITLE
[KOGITO-2550] - Fail CI if change in CRDs when running operator-sdk g…

### PR DESCRIPTION
**Pull request by Kaitou786**
…enerate crds/k8s

See: https://issues.redhat.com/browse/KOGITO-2550
Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -41,7 +41,12 @@ jobs:
       - run: go get golang.org/x/tools/cmd/cover
       - checkout
       - run: go mod tidy
-      - run: go vet ./...
+      - run:
+          name: Check Vet
```
Comment (ricardozanini) [inline .circleci/config.yml:45]: Reading again, I believe that this should be in `lint` area.

Comment (Kaitou786) [reply .circleci/config.yml:45]: Yes, I am confused about that part I'll shift this to the lint section.

**Comment by Kaitou786** [comment]
> I really like the message when it fails. @Kaitou786 I'm assuming you've tested adding/modifying the `*_types.go` files?

I tested modifying files and running that clause ` [[ -z "$changed_files" ]] ||  (printf "Change is detected in some files: \n$changed_files\n Did you run "make test" before sending the PR?" && exit 1)` it printed the msg and I believe the go-vet.sh will change the files if someone forgets to run `make test`

**Comment by Kaitou786** [comment]
> I really like the message when it fails. @Kaitou786 I'm assuming you've tested adding/modifying the `*_types.go` files?

@ricardozanini  Do you want me to set this mesage for all the checks?

**Comment by ricardozanini** [comment]
> > I really like the message when it fails. @Kaitou786 I'm assuming you've tested adding/modifying the `*_types.go` files?
> 
> @ricardozanini Do you want me to set this mesage for all the checks?

No need. :)

**Pull request merged**